addon/styles/partials/_uni-step-card.scss

```diff
@@ -5,6 +5,62 @@
         width: 100%;
     }
 
+    text-align: center;
```

Comment:
This changes should be on uniQ

Reply:
[Here](https://github.com/uniplaces/uniplaces-uniq/blob/master/src/styles/_components/_step-card.scss)